src/SvelteTable.svelte

```diff
@@ -87,14 +87,16 @@
     hoverRow = rowIdx;
   };
 
-  const sortRowsBy = (key) => {
+  const sortRowsBy = (key, descendingOverride) => {
     if (!isSortable) return;
 
     const columnData = columns.find((column) => column.key === key);
     if (columnData.sortable === false) return;
 
-    if (lastSortedKey === key) {
+    if (typeof descendingOverride === 'undefined' && lastSortedKey === key) {
       sortDescending = !sortDescending;
+    } else if (typeof descendingOverride !== 'undefined') {
+      sortDescending = descendingOverride;
```

Comment:
Not a fan of `else if` blocks and variables that are `boolean | undefined`. Also sortDescending is accessible from top-level of the component so no need to pass it it just adds confusion.

I suggest making a separate function:
```js
  const getSortingOrder = (key, override = false) => {
    if(override) return sortDescending;
    if(lastSortedKey === key) return !sortDescending;
    return false;
  }
```

and then on L96 just call 
```js
    sortDescending = getSortingOrder(key, override);
```

wyt